INSTALL.md

```diff
@@ -25,11 +25,11 @@ source code.  Unpack as follows.
     tar xzf <some dir>/accumulo-X.Y.Z-bin.tar.gz
     cd accumulo-X.Y.Z
 
-There are three scripts in the the `bin/` directory that are used to manage Accumulo:
+There are three scripts in the tarball distribution that are used to manage Accumulo:
 
-1. `accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
-2. `accumulo-service` - Runs Accumulo processes as services
-3. `accumulo-cluster` - Manages Accumulo cluster on a single node or several nodes
+1. `bin/accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
+2. `contrib/accumulo-service` - Runs Accumulo processes as services
```

Comment:
In my view, `contrib/` should be for add-ons/wrappers which could be separate projects on their own, but which we are currently maintaining to provide convenient functionality.

I think it makes sense to have a basic launch script in `bin/`, which minimally wraps the execution of java. That minimal wrapper wouldn't make sense as its own project and thus (by my opinion) shouldn't be in `contrib/`.

I'm far less opinionated on the convenience scripts for producing the example configs and for executing `make` on the unpacked native tarball. They could probably go either place.